app/views/handlers/__init__.py

```diff
@@ -38,3 +39,9 @@ def individual_response_url(
                 journey=journey,
             )
     return None
+
+
+def show_individual_response_link(location, schema):
+    section = schema.json.get("individual_response", {}).get("individual_section_id")
+    if location.block_id == schema.get_first_block_id_for_section(section):
+        return True
```

Comment:
This currently doesn't return false.
```suggestion
    return location.block_id == schema.get_first_block_id_for_section(section)
```

Reply:
Added your suggestion, thanks!